Services/Interfaces/web_interface/static/js/common/data_collector_util.js

```diff
@@ -23,14 +23,15 @@ function lock_collector_ui(lock=true){
         $("#collector_operation").hide();
     }
     $('#collect_data').prop('disabled', lock);
+    $('#stop_collect_data').prop('disabled', !lock);
 }
 
 function _refresh_status(socket){
     socket.emit('data_collector_status');
 }
 
 function update_progress(current_progress, total_progress){
-    $("#current_progess_bar_anim").css('width', current_progress+'%').attr("aria-valuenow", current_progress);
+    $("#current_progess_bar_anim").css('width', (current_progress === 0 ? 100 : current_progress)+'%').attr("aria-valuenow", current_progress);
```

Comment:
why 100% when progress is 0 ? it's a display issue ?

Reply:
when a collection is done very quickly the bar remains at 0, it does not have time to refresh. Nothing serious, then. Just to indicate that the current step is being done anyway. even if it is true that if you wait 2 minutes you see the one below move X)